lib/cli/cli.js

```diff
@@ -12,12 +12,12 @@
  */
 
 const debug = require('debug')('mocha:cli:cli');
-const symbols = require('log-symbols');
 const yargs = require('yargs/yargs');
 const path = require('path');
+const ansi = require('ansi-colors');
+const symbols = require('log-symbols');
```

Comment:
Please insert these requires to their original lines.
It's pedantic, I know, but reviewing is easier that way.

Reply:
Done.